### ISSUE TITLE
Teams connector and Iframe size

### ISSUE THREAD

**Issue by ayblanchet** [OfficeDev/microsoft-teams-library-js#2251]
Hi,
I have a problem with the new teams, the size of the iframe of my connector does not adapt to the content. I compared the code with another connector and it is not the same. How can I interact with the iframe to tell it the correct size of the connector?
Github connector:
![image](https://github.com/OfficeDev/microsoft-teams-library-js/assets/17802241/c9a30f66-5af1-43bd-815f-1d40f99d32d0)

My connector:
![image](https://github.com/OfficeDev/microsoft-teams-library-js/assets/17802241/1ddb24cf-c75f-41fb-a457-170747c9121e)

Thanks,

Best regards,

**Comment by Dinesh-MSFT** [comment]
Hi @ayblanchet - Thanks for raising the query.
We will look into it and let you know the updates.

**Comment by melsener** [comment]
Hi @Dinesh-MSFT 
We are also having the same issue. Here are the screenshots of Microsoft Teams V1 and the new teams both from the same app.
<img width="1938" alt="Screenshot 2024-04-29 at 14 41 22" src="https://github.com/OfficeDev/microsoft-teams-library-js/assets/9118234/627e5d5a-e98e-44f5-8f43-df93e56c13b7">
<img width="1925" alt="Screenshot 2024-04-29 at 14 43 40" src="https://github.com/OfficeDev/microsoft-teams-library-js/assets/9118234/db9e1699-528e-4f7b-a187-e6250133e970">

As you can see, although we're serving the same iframe, the new Teams Client renders it differently and smaller, i.e., the UI does not fit in it. We are using `@fluentui/react-northstar` `v0.50.0` for the UI part.

Although we're facing this issue, some other connectors e.g. GitHub is not affected as same as we are, you can see the below screenshot taken from the Teams V2 client:
<img width="1938" alt="Screenshot 2024-04-29 at 14 41 44" src="https://github.com/OfficeDev/microsoft-teams-library-js/assets/9118234/4d699875-81d8-4563-abff-fdc60348a826">

If you need something else from us regarding this issue, please let us know. Thanks.

**Comment by ChetanSharma-msft** [comment]
Hello @melsener - Thanks for sharing the information.
We will look into it and let you know the updates.

**Comment by Dinesh-MSFT** [comment]
Hi @melsener,
Sorry for the delay in the response. We are checking this issue internally.
 We will look into it and let you know the updates.

**Comment by melsener** [comment]
hey @Dinesh-MSFT any updates on this?

**Comment by simplypixi** [comment]
@ChetanSharma-msft @Dinesh-MSFT 
Same issue here. The iframe is very small on v2. Everything worked good on v1.

![Zrzut ekranu 2024-06-26 o 13 48 09](https://github.com/OfficeDev/microsoft-teams-library-js/assets/5683490/8f68b063-48ad-4bb8-8e02-6965d8589252)

**Comment by simplypixi** [comment]
@ChetanSharma-msft @Dinesh-MSFT 
What's the current status? You've been looking into this internally since May 6th.

**Comment by pbylina** [comment]
Any update? The issue is blocking our Teams integration.

**Comment by simplypixi** [comment]
> Hello @Everyone- Apologies for the delay in response.
> 
> We understand that there have been no updates to the repository for some time, and we appreciate your feedback.
> 
> We have received communication from our engineering team indicating that Connectors is being deprecated in Teams, and they will not be implementing any UI fixes in this area.
> 
> They recommend considering a migration to workflows for a more effective solution. [Retirement of Office 365 connectors within Microsoft Teams](https://devblogs.microsoft.com/microsoft365dev/retirement-of-office-365-connectors-within-microsoft-teams/)

Yeah, we saw that information... in July, dear Internet Explorer. :)
